org.hl7.fhir.utilities/src/main/resources/Messages.properties

```diff
@@ -380,8 +380,8 @@ Unable_to_resolve_name_reference__at_path_ = Unable to resolve name reference {0
 Details_for__matching_against_Profile_ = Details for {0} matching against profile {1}
 Does_not_match_slice_ = Does not match slice ''{0}'' (discriminator: {1})
 Profile__does_not_match_for__because_of_the_following_profile_issues__ = Profile {0} does not match for {1} because of the following profile issues: {2}
-This_element_does_not_match_any_known_slice_ = This element does not match any known slice{0}
-defined_in_the_profile = defined in the profile 
+This_element_does_not_match_any_known_slice_ = This element does not match any known slice {0}
```

Comment:
This is the only real change

The rest were whitespace changes made by github.dev